packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -465,6 +465,41 @@ describe('FLE tests', () => {
       const compactResult = await shell.executeLine(`autoMongo.getDB('${dbname}').test.compactStructuredEncryptionData()`);
       expect(compactResult).to.include('ok: 1');
     });
+
+    it('creates an encrypted collection and generates data encryption keys automatically per encrypted fields', async() => {
+      const shell = TestShell.start({ args: ['--nodb', `--cryptSharedLibPath=${cryptLibrary}`] });
+      const uri = JSON.stringify(await testServer.connectionString());
+      await shell.waitForPrompt();
+      await shell.executeLine('local = { key: BinData(0, "kh4Gv2N8qopZQMQYMEtww/AkPsIrXNmEMxTrs3tUoTQZbZu4msdRUaR8U5fXD7A7QXYHcEvuu4WctJLoT+NvvV3eeIg3MD+K8H9SR794m/safgRHdIfy6PD+rFpvmFbY") }');
+      await shell.executeLine(`keyMongo = Mongo(${uri}, { \
+        keyVaultNamespace: '${dbname}.keyVault', \
+        kmsProviders: { local } \
+      });`);
+      await shell.executeLine(`secretDB = keyMongo.getDB('${dbname}')`);
+      await shell.executeLine('var { collection, encryptedFields } = secretDB.createEncryptedCollection(\'secretCollection\', { \
+        provider: \'local\', \
```

Comment:
You don’t need to do all of this string escaping if you use template strings, like a few lines above (and you can probably also drop the line continuations at the end of each line then)

Reply:
I actually started with template strings but my eslint went havoc on me and won't let me by without this autofix. Weird but could you confirm if that is actually the case?

Reply:
Working fine now - somehow 😮